tests/foreman/ui/test_computeresource.py

```diff
@@ -348,11 +352,11 @@ def test_positive_VM_import(session, module_ca_cert, module_org,
             'provider_content.api4': version,
             'provider_content.datacenter.value': rhev_data['datacenter'],
             'provider_content.certification_authorities': module_ca_cert,
-            'locations.resources.assigned': [location.name],
+            'locations.resources.assigned': [module_loc.name],
         })
-        session.hostgroup.update(hostgroup_name, {'deploy_on': name+" (RHV)"})
+        session.hostgroup.update(hostgroup_name, {'host_group.deploy': name+" (RHV)"})
```

Comment:
I am also surprised that this ever worked. But it had to work at some point, otherwise this wouldn't even get into PR.

Looking into RP, it seems this test has never run in 6.5 and only ran (and failed) twice in 6.6.